src/uk/gov/defra/ffc/Provision.groovy

```diff
@@ -132,7 +132,7 @@ class Provision implements Serializable {
     return "$repoName-pr$pr-$queueName"
   }
 
-  static def getBuildQueueEnvVars(ctx, repoName, pr, environment) {
+  private static def getMessageQueueCreds(ctx) {
```

Comment:
In Groovy, `private` is good to help document the codes intent.  But in case you weren't aware it's not actually private.

There is an outstanding bug tracked against it that's been unresolved for 14 years.  I'm not expecting an imminent fix..

https://issues.apache.org/jira/browse/GROOVY-1875

Reply:
I didn't know it didn't do anything! As you say, as much for documentation and intent than anything.